example/oven_fsm.dart

```diff
@@ -0,0 +1,228 @@
+/// Copyright (C) 2023 Intel Corporation
```

Comment:
I think it will also be necessary to add `oven_fsm.dart` to `test/example_test.dart`.